base/applications/regedit/edit.c

```diff
@@ -1898,7 +1876,7 @@ static LONG CopyKey(HKEY hDestKey, LPCWSTR lpDestSubKey, HKEY hSrcKey, LPCWSTR l
     dwIndex = 0;
     do
     {
-        cbName = sizeof(szSubKey) / sizeof(szSubKey[0]);
+        cbName = ARRAY_SIZE(szSubKey);
```

Comment:
ARRAY_SIZE, or _countof (as you used elsewhere), or...?

Reply:
Actually _countof would be the best. But that would have to be done in many places in here. I do intend to open a separate PR about that later, which changes all at once. Until then I do use what other places of regedit do currently use, which is ARRAY_SIZE.

Reply:
That's because regedit is/was wine-synced.